prow/config.yaml

```diff
@@ -1231,7 +1231,7 @@ presubmits:
       preset-service-account: true
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180510-540e0e395-experimental
```

Comment:
What is experimental?

Reply:
it was added in https://github.com/kubernetes/test-infra/pull/6859. it seems to be used for canary jobs, but also in some real jobs?

Reply:
cc @BenTheElder

Reply:
https://github.com/kubernetes/test-infra/blob/51cf02e75d8c5ed5da3dff8ea025b270f86de3d6/images/kubekins-e2e/Makefile#L24-L29